TeamServer/Models/Managers/HttpManager.cs

```diff
@@ -225,7 +225,7 @@ private async Task GeneerateCert()
 
             // Create password for certificate protection
             var passwordForCertificateProtection = new SecureString();
-            foreach (var @char in "p@ssw0rd")
+            foreach (var @char in CertificatePassword)
```

Comment:
Use the CertificatePassword property to avoid magic values.

Reply:
looks good, thank you.